series/ostrava-pyvo/events/2018-08-01-Letni-kino-2.yaml

```diff
@@ -0,0 +1,27 @@
+city: vratimov
+start: 2018-08-01 18:00:00
+name: Ostravské Pyvo
+topic: "Letní kino #2"
+description: |
+    Druhé letní Pyvo s výletem do letního kina.
+
+    Pyvo začne jako minule v 18:00 na cyklostezce mezi Ostravou a Frýdkem v
+    Coolárně ve Vratimově. Kolem osmé večerní se společně vydáme na cca
+    15km trasu do autokempu u Olešné, kde přibližně ve 21:00 začně
+    promítání animovaného filmu Mimišéf.
+
+    Letní kino ve Frýdku-Místku:
+    http://www.frydekmistek.cz/cz/o-meste/0699038-letni-kino-kazdou-stredu-
```

Comment:
Dá se nějak udělat, aby to byly odkazy?

Reply:
Markdownem.

Reply:
Markdown standardně odkazy z URL adres nedělá, ale je na to plugin. Dalo by se to zařídt, pokud to ještě na pyvo.cz není.